centos_ci.sh

```diff
@@ -305,7 +299,7 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags=basic,openshift,version,cmd-config,profile -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags=basic,openshift,cmd-config,profile -format pretty"
```

Comment:
@budhrg Can we make this part of our Makefile so we don't have to put those stuff in here. I think here we can just have `MINISHIFT_VM_DRIVER=kvm make integration` and godog part can be in Makefile.

Reply:
@praveenkumar @budhrg This direct passing of ```GODOG_OPTS=``` will contradict the pr #1379 for sure. The pr says ```make integration``` will run the basic test. So passing the options from here will overwrite the ```GODOG_OPTS``` make sense IMO.

Reply:
> This direct passing of GODOG_OPTS= will contradict the pr #1379 for sure. The pr says make integration will run the basic test. So passing the options from here will overwrite the GODOG_OPTS make sense IMO.

If it is then, required changes need to be in the PR #1379.

Reply:
> Can we make this part of our Makefile so we don't have to put those stuff in here. I think here we can just have MINISHIFT_VM_DRIVER=kvm make integration and godog part can be in Makefile.

@praveenkumar  Should be done in separate issue. No context with current issue.

> Can we make this part of our Makefile so we don't have to put those stuff in here

There are two occasions where

Reply:
@budhrg I want to come with a question, How you handle the godog option in the make file ? are you going to hardcode the GODOG_OPTS ?

Reply:
@amitkrout , @praveenkumar , let's discuss here https://github.com/minishift/minishift/issues/1431. It has no context with current PR/issue.

Reply:
cc_ @LalatenduMohanty @agajdosi

Reply:
Why remove `version` ?

Reply:
@gbraad  Couldn't find any test with this tag? Probably it got renamed with `cmd-version`.

Reply:
@amitkrout , @agajdosi , @gbraad , updated with `cmd-version` tag.